refarch-eai/src/main/resources/application-local.yml

```
@@ -1,2 +1,2 @@
 server:
-  port: 8084
\ No newline at end of file
+  port: 8085
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)